StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/MaintainabilityRules/SA1413CodeFixProvider.cs

```diff
@@ -115,5 +121,11 @@ private static SyntaxNode RewriteAnonymousObjectInitializer(AnonymousObjectCreat
             var fixedInitializer = initializer.WithInitializers(newInitializerExpressions);
             return fixedInitializer;
         }
+
+        private static TextChange GetEnumMemberTextChange(Diagnostic diagnostic, SyntaxNode syntaxRoot)
+        {
+            var member = (EnumMemberDeclarationSyntax)syntaxRoot.FindNode(diagnostic.Location.SourceSpan);
+            return new TextChange(diagnostic.Location.SourceSpan, member.ToString() + ",");
+        }
```

Comment:
💭  The code above should be the same pattern as the code for `RewriteInitializer` and `RewriteAnonymousObjectInitializer`.
If this code performs better than the existing code, then please also change the other methods to use the same pattern, for consistency.